pkg/registry/apis/secret/register.go

```diff
@@ -24,11 +24,6 @@ func RegisterDependencies(
 		return nil, fmt.Errorf("registering access control roles: %w", err)
 	}
 
-	// TODO: Remove once the DB schema is more stable.
```

Comment:
@macabu  will make a PR for this 

I'm remving it here so we can also remove the places we need to set it explicitly (and docs)

Reply:
https://github.com/grafana/grafana/pull/110026